### ISSUE TITLE
Spyder has encountered a problem and pop up an error window

### ISSUE THREAD

**Issue by Rayaray** [spyder-ide/spyder#5044]
## Description

**What steps will reproduce the problem?**

1. Just open spyder(System info: windows 7 Enterprise service Pack 1)
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\xxxx\Documents\app\WinPython-64bit-3.6.2.0Qt5\python-3.6.2.amd64\lib\site-packages\spyder_terminal\terminalplugin.py", line 156, in __wait_server_to_start
    QMessageBox.Ok)
TypeError: 'Icon' object is not callable

## Version and main components

* Spyder Version: 3.2.1.dev0 
* Python Version: 3.6.2
* Qt Versions: 5.9.1, PyQt5 5.9 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.2.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26 (OK)
qtconsole >=4.2.0:  4.3.0 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.2 (OK)

```

**Comment by ccordoba12** [comment]
@andfoy, I think this was already fixed, right?

**Comment by andfoy** [comment]
Yes, this should be fixed as of spyder-terminal v0.2.1
